packages/eslint-config-terra/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "eslint-config-terra",
+  "version": "3.10.0",
+  "description": "Terra Eslint Config",
+  "main": "eslint.config.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/eslint-config-terra"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-config",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/eslint-config-terra/issues"
+  },
+  "engines": {
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
+  },
+  "homepage": "https://github.com/cerner/eslint-config-terra",
+  "scripts": {
+    "clean": "rm -rf node_modules",
+    "clean:install": "npm run clean && npm install",
+    "lint": "eslint --ext .js,.jsx .",
+    "postpublish": "git push --follow-tags",
+    "prepublishOnly": "npm whoami && npm run lint",
+    "release:major": "npm version major -m 'Released version %s' && npm publish",
+    "release:minor": "npm version minor -m 'Released version %s' && npm publish",
+    "release:patch": "npm version patch -m 'Released version %s' && npm publish"
+  },
+  "dependencies": {
+    "eslint-config-airbnb": "^18.0.0",
+    "eslint-plugin-compat": "^3.3.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "7.14.3",
```

Comment:
```suggestion
    "eslint-plugin-react": "7.19.0",
```

Reply:
Major version bump changes will be in another PR

Reply:
I unresolved this since we added the namespace on the package.

Reply:
Can we unlock the version or have we lost trust in the plugin releasing passive updates? I ran into this issue a few days ago. Unable to resolve the right version of the plugin that was a peer dependency of another dependency. 

^7.29.0

Reply:
personally i don't trust them

Reply:
we should update this if we're major versioning with the scoped package.